### PR TITLE
Fix Issue in debug panel.

### PR DESCRIPTION
**Pull request by jafaripur**
add getDriverName() method in MongoDB connection object. This method called in debug panel and getting error for mongodb because this method is not exist in mentioned class.
```
Exception 'yii\base\UnknownMethodException' with message 'Calling unknown method: yii\mongodb\Connection::getDriverName()' 
in /var/www/html/shop/vendor/yiisoft/yii2/base/Component.php:285
```

### CODE REVIEW

File: Connection.php [hunk 1]
```diff
@@ -276,4 +276,14 @@ protected function initConnection()
     {
         $this->trigger(self::EVENT_AFTER_OPEN);
     }
+    
+    /**
+     * Returns the name of the DB driver.
+     * by an end user.
```
Comment (samdark) [inline Connection.php]: `by an end user.`?

**Comment by cebe** [comment]
it should be fixed in debug panel to not all it instead of adding it here.

**Comment by cebe** [comment]
I have fixed it in debug panel. Thanks for you contribution, @jafaripur!

**Comment by samdark** [comment]
How to reproduce the issue?

**Comment by jafaripur** [comment]
Thanks.

**Pull request closed without merging**